chapters/extensions/VK_KHR_sampler_ycbcr_conversion.md

```diff
@@ -0,0 +1,155 @@
+# VK_KHR_sampler_ycbcr_conversion
+
+> Promoted to core in Vulkan 1.1
+
+This extension, in short, adds support to do texture sampling on YCbCr color space natively. The Vulkan spec was originally designed only for RGB color space and therefore multiple additions to the spec were added with this extension including many new formats, the concept of multi-planar formats, image aspect plane, disjoint memory, and the `VkSamplerYcbcrConversion` object.
+
+While this section will go over more of this extension in details, **disclaimer**:
+
+> The use of YCbCr sampler conversion is a niche area in computer graphics and mainly stems from the fact analog inputs (for example cameras) can read in data better this way and also it can be better for compression (example JPEG). This means it is not important for the common Vulkan developer to need to understand the additions of this extension.
```

Comment:
I'm not sure we should say it's niche in "computer graphics" - depending on how you define that. Maybe say "3D graphics" and say it's mainly used for processing inputs from video decoders and cameras?